agent/host_agent_test.go

```diff
@@ -157,6 +157,18 @@ var _ = Describe("Agent", func() {
 
 			output, _, err = runner.ExecByoDockerHost(byoHostContainer)
 			Expect(err).NotTo(HaveOccurred())
+
+			// wait until the agent process starts inside the byoh host container
+			Eventually(func() bool {
+				containerTop, _ := runner.DockerClient.ContainerTop(ctx, byoHostContainer.ID, []string{})
+				for _, proc := range containerTop.Processes {
+					if strings.Contains(proc[len(containerTop.Titles)-1], "agent") {
+						return true
+					}
+
+				}
+				return false
+			}).Should(BeTrue())
```

Comment:
Do we set an explicit timeout interval for this `eventually` block? This might fail as the default timeout is 1 sec.

Reply:
Good point. What do you think is a reasonable timeout for this? `60s`?